src/main/java/repo/impl/PostgresMigratedFormSessionRepo.java

```diff
@@ -0,0 +1,418 @@
+                    new Object[] {sessionId},
+                    new InstanceSessionMapper());
+            if(instanceSessions.size() < 1) {
+                continue;
+            }
+            sessions.add(buildSerializedSession(entrySession, instanceSessions.get(0)));
+        }
+        return sessions;
+    }
+
+    @Override
+    @Lock(LockModeType.OPTIMISTIC)
+    public <S extends SerializableFormSession> Iterable<S> save(Iterable<S> entities) {
+        for(SerializableFormSession session: entities){
+            save(session);
+        }
+        return entities;
+    }
+
+    private byte[] writeToBytes(Object object){
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        ObjectOutputStream oos;
+        try {
+            oos = new ObjectOutputStream(baos);
+            oos.writeObject(object);
+        } catch(IOException e){
+            throw new RuntimeException(e);
+        }
+        return baos.toByteArray();
+    }
+
+    @Override
+    @Lock(LockModeType.OPTIMISTIC)
+    public <S extends SerializableFormSession> S save(S session) {
+        return null;
+    }
+
+    public SerializableFormSession findOneWrapped(String id) throws FormNotFoundException {
+        try {
+            return findOne(id);
+        } catch(EmptyResultDataAccessException e) {
+            throw new FormNotFoundException(id);
+        }
+    }
+
+    @Override
+    public SerializableFormSession findOne(String id) {
+        List<EntrySession> entrySessions = this.jdbcTemplate.query(
+                replaceTableName("SELECT * FROM %s WHERE session_id = ?", POSTGRES_MIGRATED_ENTRYSESSION_TABLE_NAME),
+                new Object[] {id},
+                new EntrySessionMapper());
+
+        List<InstanceSession> instanceSessions = this.jdbcTemplate.query(
+                replaceTableName("SELECT * FROM %s WHERE sess_id = ?", POSTGRES_MIGRATED_SESSION_TABLE_NAME),
+                new Object[] {id},
+                new InstanceSessionMapper());
+
+        InstanceSession instanceSession = instanceSessions.get(0);
+        EntrySession entrySession = entrySessions.get(0);
+        return buildSerializedSession(entrySession, instanceSession);
+    }
+
+    public SerializableFormSession buildSerializedSession(EntrySession entrySession, InstanceSession instanceSession) {
+        SerializableFormSession session = loadSessionFromJson(instanceSession.getSessionJson());
+        session.setDateOpened(convertIsoToJavaDate(entrySession.getCreatedDate()));
+        session.setId(entrySession.getSessionId());
+
+        if (session.getRestoreXml() == null) {
+            session.setRestoreXml(restoreFactory.getRestoreXml());
+        }
+
+        if(session.getPostUrl() == null) {
+            session.setPostUrl(
+                    StringUtils.getPostUrl(host,
+                            session.getDomain(),
+                            entrySession.getAppId()));
+        }
+        PrototypeUtils.setupPrototypes();
```

Comment:
What does this do? It's a bit weird having a static call that doesn't take any args and doesn't return anything.

Should this not be called as part of the server setup?